ttnn/module.mk

```diff
@@ -4,15 +4,15 @@ TTNN_PYBIND11_LOCAL_SO = ttnn/ttnn/_ttnn.so
 
 TTNN_DEFINES =
 
-TTNN_INCLUDES = $(TT_EAGER_INCLUDES) $(TT_LIB_INCLUDES) -Ittnn/cpp
+TTNN_INCLUDES = $(TT_EAGER_INCLUDES) $(TT_LIB_INCLUDES) -Ittnn/cpp -Ittnn/cpp/ttnn
 TTNN_PYBIND11_INCLUDES = $(TTNN_INCLUDES) $(shell python3-config --includes) -Itt_metal/third_party/pybind11/include
 
 TTNN_LDFLAGS = -ltt_dnn -ldtx -ltensor -ltt_metal -lyaml-cpp $(LDFLAGS)
-TTNN_PYBIND11_LDFLAGS = $(TTNN_LDFLAGS) -ltt_lib_csrc  #At some point possibly add: -lttnn
+TTNN_PYBIND11_LDFLAGS = $(TTNN_LDFLAGS) -ltt_lib_csrc -lttnn
 
 TTNN_CFLAGS = $(CFLAGS) -Werror -Wno-int-to-pointer-cast -fno-var-tracking
 
-TTNN_SRCS =
+TTNN_SRCS = ttnn/cpp/ttnn/*.cpp
```

Comment:
File itself looks good - but would this wildcard capture any subdirs?

Reply:
We don't currently have any subdirectories that have *.cpp files under `ttnn/cpp/ttnn`. It's still a good idea to update the globbing since we have `*.hpp` files in one of those subdirectories. Thanks!

Reply:
updated the glob here: https://github.com/tenstorrent-metal/tt-metal/pull/6131/commits/b378ffca00eecaf3bc6596fbea8bbfe949fc346e#diff-17a8924c39fe9a52967c4c3e57d8dbab704ae48eebcbcda9d5f3665b7a839ba2R15